src/amo/routes.js

```diff
@@ -5,8 +5,7 @@ import App from './containers/App';
 import Home from './containers/Home';
 
 export default (
-  <Route path="/" component={App}>
+  <Route path="/(:lang/)" component={App}>
```

Comment:
Yes this does work as long as the rest of the URL matches. See https://github.com/mozilla/addons-frontend/pull/770/files#diff-4f380033f80b73f222c107868068167aR72